README.md

```diff
@@ -238,15 +238,41 @@ Similarly, if your application requires on-download progress logic, pytube expos
 
 pytube also ships with a tiny cli interface for downloading and probing videos.
 
-Let's start with downloading:
+Let's start with downloading (it will download the mp4 stream with the highest resolution by default):
 
 ```bash
-$ pytube http://youtube.com/watch?v=2lAe1cqCOXo --itag=22
+$ pytube https://www.youtube.com/watch?v=aqz-KE-bpKQ
```

Comment:
For consistency, please use `http://youtube.com/watch?v=2lAe1cqCOXo` as your video example.